backend/controllers/message_controller.js

```diff
@@ -0,0 +1,178 @@
+const Chat = require("../models/Chat.js");
+const ChatService = require("../services/ChatService.js");
+const User = require("../models/User.js");
```

Comment:
I think you should use UserService here like we agreed on

Reply:
Why? I'm not creating any user?